### PR TITLE
Release/0.5.0

### PR DESCRIPTION
**Pull request by anxolin**
This release:
*  Stops using Dune v0 (it will load the stats from https://raw.githubusercontent.com/cowprotocol/cow-fi/configuration/config/stats.json)
* Fixes the 404 error when you try to follow the link to Dune dashboard

## Change log

c6f8c10 Load stats from config + deprecate Dune v0 (#35)
5c6638e Merge branch 'main' into develop
ca132ec update Dune analytics url
3ec835f Replaced cowswap.exchange with swap.cow.fi (#30)

### CODE REVIEW

File: services/config.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  lastModified: Date
+}
+
+type CowStatsConfig = Omit<CowStats, 'lastModified'> & { lastModified: string }
+
+async function getFromConfig<T>(configFilePath: string): Promise<T> {
+  const response = await fetch(CONFIG_PATH + `${configFilePath}`)
+  return await response.json()
+}
```
Comment (alfetopito) [inline services/config.tsx:17]: What about some error handling in case this call/json decoding fails?

**Review by elena-zh — APPROVED** [review]
LGTM!

**Review by alfetopito — APPROVED** [review]
Approved with a comment

File: services/config.tsx (hunk 1 above)
Comment (anxolin) [reply services/config.tsx:17]: I want it to fail if there's an error. This way, it will use the old value

**Pull request merged**